src/implementation/Client/GRPCClient/workflow.ts

```diff
@@ -23,7 +24,7 @@ export default class GRPCClientWorkflow implements IClientWorkflow {
   }
 
   get(_instanceId: string, _workflowComponent?: string | undefined): Promise<WorkflowGetResponseType> {
-    throw new Error("Method not implemented.");
+    throw new MethodNotImplementedError();
```

Comment:
This can be same as GrpcNotSupportedError, I don't think we will need MethodNotImplemented error then.